apps/explorer/lib/explorer/graphql.ex

```diff
@@ -328,7 +328,7 @@ defmodule Explorer.GraphQL do
       left_join: wf in CeloWalletAccounts,
       on: tt.from_address_hash == wf.wallet_address_hash,
       left_join: wt in CeloWalletAccounts,
-      on: tt.to_address_hash == wf.wallet_address_hash,
+      on: tt.to_address_hash == wt.wallet_address_hash,
```

Comment:
what is wf and wt?

Reply:
`wt` is the alias of one table and `wf` is the alias of the second one.

Reply:
ok. a minute a go there were other changes too. did you mean to remove them?

Reply:
Yeah, I implemented a more complicated solution but then noticed the typo.